pages/people/index.html.njk

```diff
@@ -11,19 +11,20 @@ nav-index: 1
   Welcome to our new Buildit website featuring stories by and about our technical people and teams.
   News from our community, and an easy way to search the entire range of relevant job opportunities we have open.
 </p>
-<ul class="grav-c-list-cards-basic">
+<ul class="grav-c-list-cards-basic" property="schema:mainEntity" typeof="schema:ItemList">
```

Comment:
Need to ditch the `property="schema:mainEntity"` here too.

Also, if I'm not mistaken, this whole `<ul>` is identical to the one on the stories listing page. If so, would it perhaps make sense to break it out into a Nunjucks partial that both of the pages include? DRY FTW! :-)